src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1090,14 +1157,38 @@ void setOptimalIndex(int newOptimalIdx)
          */
         RawPacket[] rtpTransform(RawPacket pktIn)
         {
-            if (pktIn.getSSRCAsLong() != tl0Ssrc)
+            long localTl0Ssrc = this.tl0Ssrc;
```

Comment:
perhaps nitpicky, but can we name this something else?  either `currTl0Ssrc` or `tl0SsrcCopy`?  'local' brings up notions of 'local vs remote', as in local and remote ssrcs.

Reply:
There is `localMaxSentFrameCopy` a few lines bellow, so.. `localTl0SsrcCopy`?

Reply:
This is now in. Let me know if there's anything else that should change or if not feel free to merge or just approve.